### PR TITLE
Report queue time and execute time together always for [insights]

### PR DESCRIPTION
**Pull request by mayurbhandary**
The amount of time that a job spends in the queue and the amount of time that it takes to execute the job are reported independently. It would be good to report them together so that we don't see one without the other.

### CODE REVIEW

File: src/ripple/core/impl/JobQueue.cpp [hunk 1]
```diff
@@ -420,19 +420,27 @@ JobQueue::processTask (int instance)
...
-            perfLog_.jobStart(type, us, start_time, instance);
-            if (us >= 10ms)
-                getJobTypeData(type).dequeue.notify(us);
+            perfLog_.jobStart(type, q_time, start_time, instance);
+            
             job.doJob ();
+
+            //The amount of time it took to dequeue and execute the job
+            auto const x_time = date::ceil<microseconds>(
+                Job::clock_type::now() - job.queue_time());
```
Comment (nbougalis) [inline src/ripple/core/impl/JobQueue.cpp]: `job.queue_time()` should be `start_time` and the comment should be updated to reflect the fact that this represents the execution and not the wait time.

Comment (mayurbhandary) [reply src/ripple/core/impl/JobQueue.cpp]: Thanks. I made the changes.

File: src/ripple/app/ledger/LedgerMaster.h [hunk 2]
```diff
@@ -371,6 +371,35 @@ class LedgerMaster
     // without first wiping the database.
     LedgerIndex const max_ledger_difference_ {1000000};
 
+private:
+    struct Stats
+    {
+        template <class Handler>
+        Stats (Handler const& handler, beast::insight::Collector::ptr const& collector)
+            : hook (collector->make_hook (handler)),
+            mValidatedLedgerAge (collector->make_gauge ("LedgerMaster", "Validated_Ledger_Age")),
```
Comment (nbougalis) [inline src/ripple/app/ledger/LedgerMaster.h]: Formatting micro-nit:
```
    template <class Handler>
    Stats (Handler const& handler, beast::insight::Collector::ptr const& collector)
        : hook (collector->make_hook (handler))
        , validatedLedgerAge (collector->make_gauge ("LedgerMaster", "Validated_Ledger_Age"))
        , publishedLedgerAge (collector->make_gauge ("LedgerMaster", "Published_Ledger_Age"))
    { }
```

Also, note how I avoided the `m` prefix by using just `validatedLedgerAge` and `publishedLedgerAge`.

Comment (mayurbhandary) [reply src/ripple/app/ledger/LedgerMaster.h]: Thanks! I fixed it.

**Pull request closed without merging**